src/js/ia/IADevPipeline.js

```diff
@@ -16,6 +16,9 @@
                 var iadp;
                 iadp = Handlebars.templates.dev_pipeline(data.dev_milestones);
                 $("#dev_pipeline").html(iadp);
+
+                // 100% width
+                $(".site-main > .content-wrap").first().removeClass("content-wrap").addClass("wrap-pipeline");
```

Comment:
Yes, we agreed. Maybe a good place to note it is as a comment in the scss/css where `.content-wrap` is first defined. Like: 
```
// special note: .content-wrap is removed by pages x and y. see <files links>.
```